src/nanoemoji/svg.py

```diff
@@ -88,8 +88,7 @@ def _add_unique_gradients(
 
 # https://docs.microsoft.com/en-us/typography/opentype/spec/svg#coordinate-systems-and-glyph-metrics
 def _svg_matrix(transform: Affine2D) -> str:
-    # TODO handle rotation: units of Affine2D and svg matrix don't match
-    transform = transform._replace(d=-transform.d, f=-transform.f)
+    # TODO handle rotation: Affine2D uses radiens, svg is in degrees
```

Comment:
```suggestion
    # TODO handle rotation: Affine2D uses radians, svg is in degrees
```

Reply:
I still don't understand what this has to do with "rotation". We are only mapping two rectangles (the svg viewbox and the UPEM square), we are not rotating anything, nor dealing with angles

Reply:
A correct implementation would work even if we had rotate. Agreed that it shouldn't actually be happening.

Reply:
Work or fail; currently it would silently do the wrong thing.

Reply:
I'm still confused, sorry. You are just stringifying the Affine2D matrix as an SVG `matrix(a b c d e f)` attribute. The latter comprises just 6 floats, there are no rotation angles involved, no degrees nor radians (the spelling "radiens" is still incorrect, but it doesn't matter). If you were to decompose the 6-tuple matrix in the concatenation of its components (scale, translate, skew/rotation) then rotation itself (and the metric of rotation angles) could become an issue. But I don't see it happening here, and to me the TODO doesn't add information, only leaves me wondering. But I'll stop now.